react-app/App.jsx

```diff
@@ -1,33 +1,23 @@
-import { useState, useEffect } from 'react';
-import DropZoneFile from '@components/DropZoneFile';
-import ListImages from '@components/ListImages';
+import { useState } from 'react';
 import Gallery from '@components/Gallery';
+import XBlockEditView from '@components/XBlockEditView';
 import { GalleryContext } from '@contexts/galleryContext';
 import globalObject from '@constants/globalObject';
-import { getItemLocalStorage } from '@utils/localStorageUtils';
 
 import './App.css';
 
 const App = () => {
   const [filesToUploadList, setFilesToUploadList] = useState([]);
   const [galleryErrorMessage, setGalleryErrorMessage] = useState(null);
-  const { xblockId, isStudioView } = globalObject;
-
-  useEffect(() => {
-    const filesUploaded = getItemLocalStorage(xblockId) || [];
-    setFilesToUploadList(filesUploaded);
-  }, []);
+  const { isStudioView, isEditView } = globalObject;
```

Comment:
What's globalObject? can we use a more explicit name? Like `globalContext` perhaps?

Reply:
globalObject is the data that comes from the XBlock. It wouldn't be wise to use globalContext because it can be confusing with the context API of React

Reply:
What about globalXblockContext? or xblockContext?